v2/pkg/engine/resolve/resolvable_test.go

```diff
@@ -228,6 +228,221 @@ func TestResolvable_ResolveWithErrorBubbleUp(t *testing.T) {
 	assert.Equal(t, `{"errors":[{"message":"Cannot return null for non-nullable field 'Query.topProducts.reviews.author.name'.","path":["topProducts",0,"reviews",0,"author","name"]}],"data":{"topProducts":[{"name":"Table","stock":8,"reviews":[{"body":"Love Table!","author":null},{"body":"Prefer other Table.","author":{"name":"user-2"}}]},{"name":"Couch","stock":2,"reviews":[{"body":"Couch Too expensive.","author":{"name":"user-1"}}]},{"name":"Chair","stock":5,"reviews":[{"body":"Chair Could be better.","author":{"name":"user-2"}}]}]}}`, out.String())
 }
 
+func TestResolvable_NonNullableRootField_ApolloCompatabilityMode(t *testing.T) {
```

Comment:
Could you put it in a single test with multiple test cases?

```
TestResolvable_ApolloCompatabilityMode_Nullability(t *testing.T) {
  t.Run("NonNullableRootField", func(t *testing.T){
     //...
  })
}
```

Reply:
yes